components/app/PlaceItem.jsx

```diff
@@ -57,26 +58,28 @@ const Tag = ({ type, ...props }) => (
   </Box>
 )
 
-const PlaceInfo = ({ name, category, tags }) => (
+const PlaceInfo = ({ name, category, tags, collection, delivery }) => (
   <Box width="100%">
     <Row width="100%" justifyContent="space-between" mt={2} mb={1}>
       <CategoryName>
         {(category || '').toUpperCase()}
       </CategoryName>
       <CollectionMethod>
-        {(category || '').toUpperCase()}
+        {(delivery === "TRUE" ? "Delivery" : '')}
+        {(collection === "TRUE" && delivery === "TRUE" ? " & Collection" : collection === "TRUE" ? "Collection" : '')}
       </CollectionMethod>
     </Row>
     <PlaceName mb={2}>
-      {name}
+    <Link to={`/${_.kebabCase(name)}`} style={{ textDecoration: 'none' }}>{name}</Link>
```

Comment:
I would probably use something like `to={`/business/${_kebabCase(name)}` (need to match whatever is in `gatsby-node.js` though)

Reply:
The way it is currently set up it is http://localhost:8000/ashbys - there is no /business before it.  Do you mean we should add it?

Reply:
Yes, that's my suggestion. Up to you though. My idea is that it future proofs against a business happening to be named `/about` and conflicting with the actual `/about` page. It's more of something to consider for user-generated content/accounts though. Should be fine to stick to `/ashbys` if:

- business names are being checked manually
- there's a duplicate business name process (e.g. two Gail's become `gails`, `gails-2`)
- there's a name blacklist/renaming process (business `About` gets renamed to `about-brixton`).

Reply:
Good points! In 6adc23f, I've updated the business routes
 now to be `/business/${_kebabCase(name)`. I've also thrown an error if two businesses have the same name so the build will be blocked and someone will need to correct the spreadsheet. In the future we can add a duplicate business name to id conversion process - I guess the priority now is getting the site up and running

Reply:
Thanks so much @jamiecoe.  Yes, def just need to get something up there now!